WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -70,4 +72,23 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleDismissKeyboardCommand:(FBRouteRequest *)request
+{
+  [request.session.application dismissKeyboard];
+  NSError *error;
+  BOOL isKbdNotPresent =
+  [[[[FBRunLoopSpinner new]
+     timeout:5]
+    timeoutErrorMessage:@"The keyboard cannot be dismissed. Try to use custom approach instead."]
```

Comment:
This message is to custom. I would make check on device and make clear message on iPhones that failure is expected.